### PR TITLE
Upgrade to @ampproject/storybook 1.1.1

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-date-display/1.0/storybook/Basic.amp.js [hunk 1]
```diff
@@ -18,16 +18,22 @@ import * as Preact from '../../../../src/preact';
...
+  .addParameters({
+    extensions: [
+      {name: 'amp-date-display', version: '1.0'},
+      {name: 'amp-mustache', version: '0.1'},
+    ],
+    experiments: ['amp-date-display-bento'],
+  })
   .add('responsive', () => {
-    const datetime = date('Date/Time', new Date());
+    const datetime = new Date(date('Date/Time', new Date())).toISOString();
```
Comment (caroqliu) [inline extensions/amp-date-display/1.0/storybook/Basic.amp.js:36]: Can you `new Date(new Date))` in the default case? Why is the outer one necessary?

Comment (dvoytenko) [reply extensions/amp-date-display/1.0/storybook/Basic.amp.js:36]: The weird thing is that `date()` knob returns milliseconds value ¯\_(ツ)_/¯. I think we tripped over this almost in all places we used it :)

**Comment by dvoytenko** [comment]
@krdwan 

> One question - does preact-env also need to be udpated?

No. Preact just uses the standard addon. Only AMP specific addoc has been updated. See https://github.com/ampproject/storybook-addon-amp

**Comment by krdwan** [comment]
> @krdwan
> 
> > One question - does preact-env also need to be udpated?
> 
> No. Preact just uses the standard addon. Only AMP specific addoc has been updated. See https://github.com/ampproject/storybook-addon-amp

Okay sounds good, thanks for the clarification!

**Pull request merged**